jasmine/new.json

```diff
@@ -0,0 +1,10 @@
+[{
```

Comment:
### Nitpick

I already gave feedback on how to  structure multi-object Arrays, why is this still happening?